Add Q^ω, as a better distinguishment of Erdős space (#1144)
pi-base/data: spaces/S000146/properties/P000066.md

---
space: S000146
property: P000066
value: false
---

{S28} is homeomorphic to $\mathbb Z^\omega$, which can be viewed as a closed subspace of {S146}.

> It follows from {S28|P66} and the fact that the closed subspace of a {P66} space is {P66}.

Reviewer: ```suggestion
It follows from {S28|P66} and the fact that {P66} is preserved by closed subspaces.
```
